clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,116 @@
+automanagedNamespaces: 3
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: TuningSet100
+    objectBundle:
+    - basename: rc1
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 30
+- measurements:
+  - Identifier: WaitForRunningPods1
+    Method: WaitForRunningPods
+    Params:
+      desiredPodCount: 90
+      labelSelector: app = pause
+      timeout: 2m
+- measurements:
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: stop
+      label: Pods creation
+  - Identifier: TimerAdditionals
```

Comment:
Also please give more meaningful names to these timers - so that it's easy to understand what they're for.